_posts/2021-10-25-g2p-applications.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "Applications for g2p, an example with ReadAlongs"
+author: AidanPine
+categories: []
+tags: [ tutorial, explanation, tech-showcase, g2p ]
+image: assets/images/nikikw.png
+description: "Applications for g2p, an example with ReadAlongs"
+featured: false
+hidden: false
+---
+
+This is the 5th post in a seven-part series about g2p. In this post, we discuss using g2p to create interactive text/audio books with ReadAlong Studio.
+
+### G2P Blog Series Index
+
+- [Background]({{ "g2p-background" | absolute_url }})
+- [How to write a basic mapping in G2P Studio]({{ "g2p-basic-mappings-gui" | absolute_url }})
+- [Writing mappings on your computer]({{ "g2p-basic-mappings-local" | absolute_url }})
+- [Advanced mappings]({{ "g2p-advanced-mappings" | absolute_url }})
+- ReadAlong Studio & Other Applications
+- [Preprocessing mappings]({{ "g2p-preprocess" | absolute_url }})
+- [Contributing]({{ "g2p-contributing" | absolute_url }})
+
+This post will discuss the use of `g2p` in the [ReadAlongs](#readalongs) project for creating interactive audio/text documents.
+
+## Other Applications
+
+There are number of different software tools that are already making use of `g2p`. For general purpose use in Python, have a look at [this post]({{ "g2p-advanced-mappings" | absolute_url }}).
+
+### Convertextract
+
+Convertextract is a tool that lets you use `g2p` mappings to convert text inside a variety of different documents, including Microsoft Office documents like Powerpoint and Word coduments. Have a look at the [convertextract repo readme](https://github.com/roedoejet/convertextract) or at Fineen Davis' [blog post]({{ "convertextract-app" | absolute_url }}) about the Convertextract GUI.
+
+### ReadAlongs
+
+ReadAlongs is a research project from the [National Research Council's Indigenous Language Technology Project](https://nrc.canada.ca/en/research-development/research-collaboration/programs/canadian-indigenous-languages-technology-project).
+
+Language revitalization communities have potentially many different recordings of language and some associated text, but mobilizing these materials into something that is educationally useful can be time consuming. It's also tough from the learner's perspective to try and follow along the text (maybe a Word Document) just by simply playing the audio. It's easy to get lost in the recording quickly, and from personal experience, trying to do that usually involves a lot of frustrating rewinding.
```

Comment:
"Language revitalization communities" is a funny term to me. "Communities doing language revitalization"? to me, the community is pre-defined, typically an Indigenous one, by their own identify. "language revitalization communities" seems to suggest their lang revit efforts define who they are, and that feels backwards.  If you're intended to refer to communities of practice around lang revit, I'd spell it out too.